### PR TITLE
Reformat repo to clang-format 7.0 rules

### PR DESCRIPTION
**Pull request by Techjar**
This is to avoid random format conflicts in later PRs. This will probably fail lint until #8082 is merged, so do that first.

### CODE REVIEW

**Comment by Techjar** [comment]
Pinging @lioncash for thoughts on `FixNamespaceComments`.

**Comment by lioncash** [comment]
Looks good to me! Especially with keeping the trailing namespace comments consistent.

File: Source/Core/Common/MD5.cpp [hunk 1]
```diff
@@ -49,4 +49,4 @@ std::string MD5Sum(const std::string& file_path, std::function<bool(int)> report
 
   return output_string;
 }
-}
\ No newline at end of file
+}  // namespace MD5
```
Comment (Warepire) [inline Source/Core/Common/MD5.cpp]: Missing new line.

**Comment by BhaaLseN** [comment]
I generally like what I see.

One thing tho:
Can we revert #4606 (perhaps check whether `clang-format -dump-config` with `BreakBeforeBraces: Allman` includes `AfterUnion` or not) with this?

**Comment by Techjar** [comment]
Nope, `Allman` still does not set `AfterUnion`. I'm guessing this is to maintain default behavior from when `Allmen` was initially added.

**Pull request merged**